src/Program.cs

```diff
@@ -38,15 +38,17 @@ public static void Main(string[] args)
             }
 
             var searchOption = resizeRecursively ? SearchOption.AllDirectories : SearchOption.TopDirectoryOnly;
-            var validExtensions = new[] { ".jpg", ".jpeg", ".png", ".bmp", ".gif", ".tif", ".tiff" };
+            var validExtensions = new[] { ".jpg", ".jpeg", ".png", ".bmp", ".gif" };
```

Comment:
did ImageSharp lose TIFF support? not a big deal, I don't think it's that common